aws/data_source_aws_elasticache_replication_group.go

```diff
@@ -106,6 +112,7 @@ func dataSourceAwsElasticacheReplicationGroupRead(d *schema.ResourceData, meta i
 		d.Set("primary_endpoint_address", rg.NodeGroups[0].PrimaryEndpoint.Address)
 	}
 	d.Set("number_cache_clusters", len(rg.MemberClusters))
+	d.Set("member_clusters", flattenStringList(rg.MemberClusters))
```

Comment:
When using `d.Set()` with aggregate types (TypeList, TypeSet, TypeMap), [we should perform error checking to prevent issues where the code is not properly able to set the Terraform state](https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types). e.g.

```go
if err := d.Set("member_clusters", flattenStringList(rg.MemberClusters)); err != nil {
  return fmt.Errorf("error setting member_clusters: %s", err)
}
```

The same applies to the resource.